src/py42/clients/cases.py

```diff
@@ -0,0 +1,185 @@
+from datetime import datetime
+
+from py42.util import get_attribute_keys_from_class
+from py42.util import parse_timestamp_to_milliseconds_precision
+
+
+class CaseStatus(object):
+    OPEN = u"OPEN"
+    CLOSED = u"CLOSED"
+
+    @staticmethod
+    def choices():
+        return get_attribute_keys_from_class(CaseStatus)
+
+
+class CasesClient(object):
+    """A client to expose cases API.
+
+    `Rest documentation <https://developer.code42.com/api/#tag/Cases>`__
+    """
+
+    def __init__(self, cases_service, cases_file_event_service):
+        self._cases_service = cases_service
+        self._file_events = cases_file_event_service
+
+    @property
+    def file_events(self):
+        """A collection of methods for managing file events associated with a given case.
+
+        Returns :
+            :class:`py42.services.casesfileevents.FileEventsService`
```

Comment:
This link is not working. I believe it should be `py42.services.casesfileevents.CasesFileEventsService`

Reply:
Changed.